rfcs/text/0045-additional-vulnerability-fields.md

```diff
@@ -0,0 +1,138 @@
+| `vulnerability.patch.name` | text | Name of the patch |
+| `vulnerability.patch.code` | keyword | Associated patch code for example ESA-2020-13 |
+| `vulnerability.evidence` | text | A demonstration of the validity of a vulnerability claim, e.g. app.any.run replaying the exploitation of the vulnerability. |
+| `vulnerability.status` | keyword | The status field helps security teams track vulnerabilities, prioritize actions, and communicate their progress effectively. Examples- open/ignored/patched/mitigated/false_positive/risk_accepted/reopened..|
+| `vulnerability.tags` | keyword | This is different from cloud provider assigned resource tags, this is specifically for vulnerability. Vulnerability tags serve as a way to add custom metadata to vulnerabilities, enhancing their context and aiding in search and automation. |
+| `vulnerability.first_found` | date | First time a vulnerability was found on the asset. Date format similar to base field of @timestamp example format- 2016-05-23T08:05:34.853Z |
+| `vulnerability.last_found` | date | Last time a vulnerability was found on the asset. Date format similar to base field of @timestamp example format- 2016-05-23T08:05:34.853Z |
+| `vulnerability.last_scanned` | date | Last time a scan was performed on the asset. It's important as some companies are scanning on a quarterly basis. If lastfound and lastscanned are close, it means it's still an active vulnerability. Date format similar to base field of @timestamp example format- 2016-05-23T08:05:34.853Z|
+| `vulnerability.age` | int | Numbers of days since the vulnerability is active. It should be dynamically calculated (runtime fields, ingest, ...). It could either be then difference between the lastfound date and the published date (preferred). It could also be the difference between the firstfound and lastfound dates. |
+| `vulnerability.uid` | keyword | It's extremely important to be able to deduplicate different scans. It's often that we have different scanners showing the same vulnerability on the same asset. |
+| `vulnerability.type` | keyword | To conclude if the vulnerability is confirmed or potential. |
+| `vulnerability.exploitability.*` | object | - |
+| `vulnerability.exploitability.exploited` | boolean | To indicate if the vulnerability has been exploited or not. |
+| `vulnerability.exploitability.reference` | keyword | Exploitability databse for example CSA-KEV. |
+| `vulnerability.exploitability.confidence` | keyword | Confidence measure the credibility of existence and exploitability. |
+| `vulnerability.exploitability.first_seen` | date | First time of exploitability |
+| `vulnerability.exploitability.last_seen` | date | Last time of exploitability |
```

Comment:
these dates should also have time format

Reply:
Making suggested changes to include time format.